extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -263,7 +263,7 @@ var CreatePage = {
 
 	redLinkClick: function( e, titleText ) {
 		'use strict';
-		var title = new mw.Title.newFromText( titleText ),
+		var title = new mw.Title.newFromText( decodeURIComponent( titleText ) ),
```

Comment:
@inez, @mari-mcmurtrie: is the a case where `titleText` could be `undefined`? If yes, are we OK with `decodeURIComponent` throwing `Uncaught ReferenceError` or should it be handled in the code?